README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
```

Comment:
How will feedback be provided?
What type of feedback will be provided?
What will the expectations be of the committees to respond to the feedback

Reply:
These are process decisions that do not need to be included in the charter and need to be determined in course by the actual group once it is formed.